driver-core/src/main/java/com/datastax/driver/core/DataType.java

```diff
@@ -503,6 +503,19 @@ public static CollectionType frozenMap(DataType keyType, DataType valueType) {
     }
 
     /**
+     * Returns a Duration type, introduced in Cassandra 3.10.
+     * <p/>
+     * Note that a Duration type does not have a native representation in CQL, and
+     * technically, is merely a special {@link DataType#custom(String) custom type}
+     * from the driver's point of view.
+     *
+     * @return the Duration type.
+     */
+    public static DurationType duration() {
+        return DurationType.instance;
```

Comment:
We might want to add a special case in `DataType.decode()` to return this singleton as well.

Currently it returns a `CustomType` instance that is not a `DurationType`. It appears for example in the column definitions of a prepared statement.